### DISCUSSION TITLE
[Question]: Added API access 404

### DISCUSSION THREAD

**Discussion by Kevin-free** [danny-avila/LibreChat#2791] [Q&A] [answered]
### What is your question?

I added a new API with reference to the project structure, but access 404, how to troubleshoot and solve it? Is there any script command that needs to be executed?

### More Details

Add the `checkout` route in `api/server/routes/index.js`:

```javascript
const checkout = require('./checkout');

module.exports = {
 checkout,
};
```

Add code in ``api/server/routes/checkout.js`:`:

```javascript
const express = require('express');
const router = express.Router();
const checkoutController = require('../controllers/Checkout');
const { requireJwtAuth } = require('../middleware/');

router.post('/', requireJwtAuth, checkoutController);

module.exports = router;
```

### Screenshots

<img width="376" alt="image" src="https://github.com/danny-avila/LibreChat/assets/26422757/1df00948-2e0b-406b-8061-85223b0c6737">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
See `api/server/index.js` to see how routes are added

**Reply by Kevin-free** [reply]
It worked, thank you!
